src/main/java/edu/ucla/library/libcal/verticles/MainVerticle.java

```diff
@@ -110,7 +110,7 @@ private Future<HttpServer> configureServer(final JsonObject aConfig) {
             // Empty-path router to handle the variable-format calls to ProxyHandler
             router = routeBuilder.createRouter();
             router.allowForward(AllowForwardHeaders.X_FORWARD);
-            // Add body handler so we can retirve incoming body from Apps client request
+            // Add body handler so we can retreve incoming body from Apps client request
```

Comment:
I think this is supposed to be "retrieve"?

Reply:
Fixed